src/tests/ftest/container/delete.py

```diff
@@ -102,13 +105,24 @@ def test_container_delete(self):
                 cont_uuid = uuid.uuid4()
 
             self.container.destroy(force=force, poh=poh, con_uuid=cont_uuid)
+
+            # close container handle, release a reference on pool in client lib
+            # Otherwise test will ERROR in tearDown (pool disconnect -DER_BUSY)
+            if opened:
+                self.container.close()
```

Comment:
As this is repeated on lines 124-125, it should probably be put in a `finally` clause.